src/WebJobs.Script.WebHost/WebScriptHostHttpRoutesManager.cs

```diff
@@ -76,23 +60,46 @@ private void InitializeHttpFunctions(IEnumerable<FunctionDescriptor> functions,
 
                     WebJobsRouteBuilder builder = function.Metadata.IsProxy ? proxiesRoutesBuilder : routesBuilder;
                     builder.MapFunctionRoute(function.Metadata.Name, route, constraints, function.Metadata.Name);
+
+                    LogRouteMap(routesLogBuilder, function.Metadata.Name, route, httpTrigger.Methods, function.Metadata.IsProxy, _httpOptions.Value.RoutePrefix);
                 }
             }
 
             IRouter proxyRouter = null;
             IRouter functionRouter = null;
 
-            if (proxiesRoutesBuilder.Count > 0)
+            if (routesBuilder.Count == 0 && proxiesRoutesBuilder.Count == 0)
             {
-                proxyRouter = proxiesRoutesBuilder.Build();
+                routesLogBuilder.AppendLine("No HTTP routes mapped");
             }
-
-            if (routesBuilder.Count > 0)
+            else
             {
-                functionRouter = routesBuilder.Build();
+                if (proxiesRoutesBuilder.Count > 0)
+                {
+                    proxyRouter = proxiesRoutesBuilder.Build();
+                }
+
+                if (routesBuilder.Count > 0)
+                {
+                    functionRouter = routesBuilder.Build();
+                }
             }
 
             _router.AddFunctionRoutes(functionRouter, proxyRouter);
+
+            ILogger logger = _loggerFactory.CreateLogger("Host.HttpRoutes");
```

Comment:
create typed logger: ` _loggerFactory.CreateLogger<WebScriptHostHttpRoutesManager>`

Reply:
This will not make it into the table the way we need. That's what I wanted to do. Had a chat with @brettsam and we'll be working on that.

Reply:
Sorry, I misunderstood you. I thought you meant from a function app. If you use typed logger here, it will go into Kusto b/c it starts with `Microsoft.Azure.Webjobs`. I'd recommend changing that out really quick. I don't want to introduce more "Host." routes if we don't have to.